evalml/tests/data_checks_tests/test_highly_null_data_check.py

```diff
@@ -96,41 +99,16 @@ def test_highly_null_data_check_input_formats():
                                       message_code=DataCheckMessageCode.HIGHLY_NULL,
                                       details={"column": 2}).to_dict()],
         "errors": [],
-        "actions": []
+        "actions": [DataCheckAction(DataCheckActionCode.DROP_COL, details={"column": 0}).to_dict(),
+                    DataCheckAction(DataCheckActionCode.DROP_COL, details={"column": 1}).to_dict(),
+                    DataCheckAction(DataCheckActionCode.DROP_COL, details={"column": 2}).to_dict()]
     }
+    #  test Woodwork
+    ww_input = ww.DataTable(pd.DataFrame([[None, None, None, None, 0], [None, None, None, "hi", 5]]))
+    assert highly_null_check.validate(ww_input) == expected
 
     #  test 2D list
-    assert highly_null_check.validate([[None, None, None, None, 0], [None, None, None, "hi", 5]]) == {
```

Comment:
Just cleaning up. We were using the same `expected` value for each of these cases.